src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,228 @@
+    ----------
+    erwerbsm_rente_zugangsfaktor
+        See :func:`erwerbsm_rente_zugangsfaktor`.
+    entgeltp_erwerbsm_rente
+        See :func:`entgeltp_erwerbsm_rente`.
+    rentenwert
+        See :func:`rentenwert`.
+    rentenartfaktor_erwerbsm_rente
+        See :func:`rentenartfaktor_erwerbsm_rente`.
+    ges_rente_vorauss_erwerbsm
+        See :func:`ges_rente_vorauss_erwerbsm`.
+    Returns
+    -------
+    Erwerbsminderungsrente (public disability insurance claim)
+
+    """
+
+    if ges_rente_vorauss_erwerbsm:
+        out = (
+            entgeltp_erwerbsm_rente
+            * erwerbsm_rente_zugangsfaktor
+            * rentenwert
+            * rentenartfaktor_erwerbsm_rente
+        )
+    else:
+        out = 0.0
+    return out
+
+
+def ges_rente_vorauss_erwerbsm(
+    erwerbsgemindert: bool,
+    m_pflichtbeitrag: float,
+    ges_rente_wartezeit_5: bool,
+) -> bool:
+    """
+    Eligibility for Erwerbsminderungsrente (public disability insurance claim).
+
+    Legal reference: § 43 Abs. 1  SGB VI
+
+    Parameters
+    ----------
+    erwerbsgemindert
+        See basic input variable :ref:`erwerbsgemindert <erwerbsgemindert>.
+    m_pflichtbeitrag
+        See basic input variable :ref:`m_pflichtbeitrag <m_pflichtbeitrag>.
+    ges_rente_wartezeit_5
+        See :func:`ges_rente_wartezeit_5`.
+    Returns
+    -------
+    Eligibility for Erwerbsminderungsrente (public disability insurance claim) as a bool
+    """
+
+    anspruch_erwerbsm_rente = (
+        erwerbsgemindert and ges_rente_wartezeit_5 and m_pflichtbeitrag >= 36
+    )
+
+    return anspruch_erwerbsm_rente
+
+
+def entgeltp_erwerbsm_rente(
+    entgeltp: float,
+    durchschn_entgeltp: float,
+    erwerbsm_rente_params: dict,
+    age_of_retirement: float,
+) -> float:
+    """Calculating Entgeltpunkte for Erwerbsminderungsrente
+    (public disability insurance)
+    In the case of the public disability insurance,
+    pensioners are credited with additional earning points.
+    They receive their average earned income points for
+    each year between their age of retirement and the "zurechnungszeitsgrenze".
+
+    Parameters
+    ----------
+    entgeltp
+        See basic input variable :ref:`entgeltp <entgeltp>
+    durchschn_entgeltp
+        See :func:`durchschn_entgeltp`.
+    erwerbsm_rente_params
+        See params documentation :ref:`erwerbsm_rente_params <erwerbsm_rente_params>.
+    age_of_retirement
+        See :func:`age_of_retirement`.
+
+    Returns
+    -------
+    Final pension points for Erwerbsminderungsrente (public disability insurance)
+
+    """
+    zurechnungszeitsgrenze = erwerbsm_rente_params["zurechnungszeitsgrenze"]
+
+    out = entgeltp + (
+        (zurechnungszeitsgrenze - (age_of_retirement)) * durchschn_entgeltp
+    )
+
+    return out
+
+
+def durchschn_entgeltp(
+    entgeltp: float,
+    age_of_retirement: float,
+) -> float:
+    """Calculation of average earning points as part of the "Grundbewertung".
```

Comment:
```suggestion
    """Average earning points as part of the "Grundbewertung".
```

Please go through docstrings and get rid of unnecessary verbs in the beginning. It is obvious that we are doing calculations in each function *(yes, I know lots of old functions still have that...)*